diff -urN libidn-1.25/configure.ac libidn-1.25-patched/configure.ac
--- libidn-1.25/configure.ac	2012-05-22 19:26:12.000000000 +0200
+++ libidn-1.25-patched/configure.ac	2012-06-13 06:24:11.878967495 +0200
@@ -23,7 +23,7 @@
 AC_CONFIG_AUX_DIR([build-aux])
 AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_HEADERS(config.h)
-AM_INIT_AUTOMAKE([1.10 -Wall -Werror -Wno-override])
+AM_INIT_AUTOMAKE([1.10 -Wall -Wno-override])
 AM_SILENT_RULES([yes])
 
 # Library code modified:                              REVISION++
@@ -42,6 +42,9 @@
 DLL_VERSION=`expr ${LT_CURRENT} - ${LT_AGE}`
 AC_SUBST(DLL_VERSION)
 
+m4_pattern_allow([AM_PROG_AR])
+AM_PROG_AR
+
 # We can't replace this with LT_INIT([win32-dll]) yet.  For example,
 # the Ubuntu 8.04 LTS is still shipping a libtool version that doesn't
 # have it.
